### ISSUE TITLE
Home -> Header Missing Toggle Control Semantics

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#760]
When a user is signed in, the “Sign In” link in the navigation menu is replaced with a toggle control.  It can be used to expand and collapse a short sub-menu of account related controls and information, but is missing the required semantics to communicate its purpose to screen reader users. 
 
• The toggle is implemented as a < a > element, but because it has no href attribute a screen reader will not recognize it as an actionable control at all.  Regardless of this fact, the control should be a button as it has a same-page action. 

• While an aria-expanded attribute is applied to the element, it is negated by the fact that a screen reader will not recognize the outer < a > (see the previous point).  Users will be unaware that this control is a toggle and unable to gauge its current state. 
 
In addition to the above, the accessible label (provided by the alt attribute of a child < img >) only contains the single word “Avatar”.  Users aren’t even able to guess at the control’s purpose because the label text is completely inadequate. 
 
Suggested Solutions 
 
Use a native < button > element instead of a < a >.  We occasionally recommend that the role of a control can be overridden using ARIA, but there is no reason to use a < a > instead of a < button > here. 
 
Update the alt attribute of the child < img > to give this control a clear label which accurately describes its purpose.  For example, “Account Controls”, “Account Menu”, “User Profile”, etc. 
 
Proposed Code Sample 
 
File: src/components/Home/components/Header/index.js 
 
`<button className="nav-link dropdown-toggle" id="navbarDropdownMenuLink-avatar" data-toggle="dropdown" aria-expanded="false">  <img src={userProfile.profileImage} className="rounded-circle z-depth-0" alt= {Locales.strings.account_menu} /> </button> `

**Comment by rupeshparab** [comment]
Dropdown type was fixed in https://github.com/benetech/MathShare/issues/740
I have fixed the alt text

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/bbb07991d451f8217625f494fe39fc298ca62b27.
